Assets/Scripts/MVC/EnemyTank/EnemyTankView.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyTankView : MonoBehaviour
```

Comment:
Please use update, fixed update of the tank here and pass the information to the controller for the same